### PR TITLE
Bugfix estimate km

### PR DESCRIPTION
**Pull request by rebecca-albrecht**

### CODE REVIEW

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]
Nice catch, but you are repeating the same testing in two different pull requests. Can you update in one and then OK for me to merge this one.

File: tests/testthat/test-estimate_KM.R [hunk 1]
```diff
@@ -18,12 +18,12 @@ library(survival)
 #' T1.5 An error when `data` is NULL
 #' T1.6 An error when `data` does not exist in the global environment
 #' T2. The function relies on the presence of two numeric variables, AVAL and CNSR, to be present in `data`
-#' T2.1 An error when colname specified as AVAL is not present in `data`
-#' T2.2 An error when colname specified as AVAL is not numeric
-#' T2.3 An error when AVAL is not called AVAL is not numeric
-#' T2.4 An error when colname specified as CNSR is not present in `data`
-#' T2.5 An error when colname specified as CNSR is not numeric
-#' T2.6 An error when CNSR is not called CNSR is not numeric
+#' T2.1 An error when colname specified as analysis value is not present in `data`
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:21]: keep AVAL

File: tests/testthat/test-estimate_KM.R [hunk 2]
```diff
@@ -18,12 +18,12 @@ library(survival)
...
+#' T2.2 An error when colname specified as analysis value is not numeric
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:22]: Keep AVAL

File: tests/testthat/test-estimate_KM.R [hunk 3]
```diff
@@ -18,12 +18,12 @@ library(survival)
...
+#' T2.3 An error when `AVAL` is not the actual column used as analysis value and this column is not numeric
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:23]: It is still unclear. Can't you keep AVAL is not numeric. Does not really matter how the column is called. You are testing the paramter => use parameter as column => is corresponding column numeric

File: tests/testthat/test-estimate_KM.R [hunk 4]
```diff
@@ -18,12 +18,12 @@ library(survival)
...
+#' T2.4 An error when colname specified as censor value is not present in `data`
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:24]: keep CNSR

File: tests/testthat/test-estimate_KM.R [hunk 5]
```diff
@@ -18,12 +18,12 @@ library(survival)
...
+#' T2.5 An error when colname specified as censor value is not numeric
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:25]: Keep CNSR

File: tests/testthat/test-estimate_KM.R [hunk 6]
```diff
@@ -18,12 +18,12 @@ library(survival)
...
+#' T2.6 An error when `CNSR` is not the actual column used as censor value and this column is not numeric
```
Comment (SHAESEN2) [inline tests/testthat/test-estimate_KM.R:26]: See AVAL comment

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]
If you prefer we can have a call.

**Comment by rebecca-albrecht** [comment]
Ok, I guess I don't get it Steven, I am sorry! Do you mean that the tests of the estimate_km are in both branches?

Unfortunately, I cannot have a call now. I have to pick up the kids. I will check in again later tonight!

**Comment by SHAESEN2** [comment]
Still updates required. I will create a new pull request.

**Pull request closed without merging**